src/e2e/specs/auth.spec.ts

```diff
@@ -28,7 +28,7 @@ test.describe(`${process.env.E2E_TEST_ENV} - Authentication flow verification @s
 
     // assert successful login
     // will poll for url as Fxa takes some time
-    expect(page.url()).toBe(`${process.env.E2E_TEST_BASE_URL}/user/dashboard`);
+    expect(page.url()).toBe(`${process.env.E2E_TEST_BASE_URL}/user/welcome`);
```

Comment:
I don't know if it's problematic or if it's just outdated test runs or something, but there are a lot of red annotations here :sweat_smile: